v19.2/movr.md

```diff
@@ -0,0 +1,85 @@
+---
+title: MovR
+summary: The MovR application uses CockroachDB to store information about vehicles, users, and rides.
+toc: true
+---
+
+MovR is a fictional vehicle-sharing company created to demonstrate CockroachDB's features.
+
+## Overview
+
+The MovR example consists of three components:
+
+- The `movr` database, which contains several tables that hold vehicle, user, and ride data. The `movr` database is built into `cockroach demo` and `cockroach workload`.
+- The `movr` dataset, which consists of rows of data that populate tables in the `movr` database. The `movr` dataset is built into `cockroach demo` and `cockroach workload`.
+- The MovR application is a fully-functional vehicle-sharing application, written in Python. All of MovR application source code is open-source, and available on the [movr](https://github.com/cockroachdb/movr) GitHub repository.
+
+## The `movr` Database
+
+Three tables in the `movr` database store most of the user, vehicle, and ride data for MovR:
+
+- `users` represents the people registered for the service.
+- `vehicles` represents the pool of vehicles for the service.
+- `rides` represents when and where users have rented a vehicle.   
+
+Each table has a composite primary key of `city` and `id`, with `city` being first in the key.
+
+<img src="{{ 'images/v19.2/geo-partitioning-schema.png' | relative_url }}" alt="Geo-partitioning schema" style="max-width:100%" />
+
+The `movr` database also contains the `promo_codes`, `user_promo_codes`, and `vehicle_location_histories` tables.
+
+## Using `cockroach` Commands to Generate Schemas and Data for MovR
+
+Use `cockroach` commands to load the `movr` database and dataset that are built into the CockroachDB binary.
```

Comment:
More of a style thing, but I'd probably edit this slightly to say "Use the following cockroach commands ..." and then make the following 2 items a bulleted list.

Reply:
Done!